src/target/riscv/riscv-013.c

```diff
@@ -913,10 +918,21 @@ static int register_read_abstract(struct target *target, uint64_t *value,
 	return ERROR_OK;
 }
 
-static int register_write_abstract(struct target *target, uint32_t number,
-		uint64_t value, unsigned size)
+static unsigned int register_size(struct target *target, enum gdb_regno number);
+
+static int register_read_abstract(struct target *target, riscv_reg_t *value,
+		enum gdb_regno number)
+{
+	const unsigned int size = register_size(target, number);
+
```

Comment:
Detail: This blank line can be removed.

```suggestion
```

Reply:
After removing this line and running `checkpatch`:
```
WARNING: Missing a blank line after declarations
#91: FILE: src/target/riscv/riscv-013.c:927:
+       const unsigned int size = register_size(target, number);
+       return register_read_abstract_with_size(target, value, number, size);
```
So, if it's not crucial, I'd like to leave it as-is.

Reply:
Ah, I didn't know checkpatch requires blank line. 

So yes, please feel free to leave the code as is.